frontend/packages/ui/src/Table.tsx

```diff
@@ -259,6 +258,9 @@ export const Column = styled.td<{
 }>`
   ${({ width, isFirst, isLast, variant }: any) => {
     return `
+    overflow: hidden;
```

Comment:
We have TruncatedText in Table, try using it inside the `Cell` first.

Reply:
Done